### ISSUE TITLE
Usage within a ng-template

### ISSUE THREAD

**Issue by flohw** [ng-select/ng-select#1586]
**Describe the bug**
ng-select doesn't dropdown on click when placed inside an ng-template

**Reproducbile example**
[Example](https://stackblitz.com/edit/angular-w74m5n) here. There is both versions : first one with ng-template the second without. The first one is exactly the same as my real app.
Please not that add and remove buttons doesn't work as it's not for this feature I post this issue.

**Expected behavior**
The ng-select dropdown should appear. The only possible thing to do for now is to unselect the option with the cross.

**Desktop (please complete the following information):**
 - OS: archlinux
 - Browser firefox 74, chrome 80

**Smartphone (please complete the following information):**
Not tested, not applicable for my situation, and not able to test, sorry.

**Additional context**
The idea is to create a line of three inputs I can replicate as many time as I want. Same principle as an ingredient list: add button adds a line of two linked inputs: quantity on ingredients available in the list to pick one of them.

Hopefully everything is clear. Let me know if I need to clarify some points.

Thank you for this amazing component ! :slightly_smiling_face:

**Comment by flohw** [comment]
It's not the component fault but my knowledge about Angular :slightly_smiling_face: 

It was due to an infinite loop of event between ng-select onInit and the ngTemplate.

I gave the issue somewhere else and I received the solution: when adding the ngForTrackBy directive to the ngTemplate line 4 and 11 on the previous example, I don't get the loop anymore and the list is displayed properly.

The only problem I have now is that the select width is computed too small. I don't have search issues for that at the moment. I will close the issue once I fix that last problem.

Thank you and I hope the solution can help someone else.
